### PR TITLE
(PC-10293) Revert search sanitize query params

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10293

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/features/home/components/OffersModule.tsx [hunk 1]
```diff
@@ -43,6 +42,7 @@ export const OffersModule = (props: OffersModuleProps) => {
       const timestampsInMillis = item.offer.dates?.map((timestampInSec) => timestampInSec * 1000)
       return (
         <OfferTile
+          key={item.objectID}
```
Comment (antoinewg) [inline src/features/home/components/OffersModule.tsx:45]: `keyExtractor` suffit pas ?

File: src/features/search/pages/__tests__/reducer.test.ts [hunk 2]
```diff
@@ -55,6 +55,21 @@ describe('Search reducer', () => {
...
+      priceRange: [30, 500],
+    }
+    const action: Action = {
+      type: 'INIT_FROM_SEE_MORE',
+      payload: parameters as Partial<SearchState>,
+    }
+    expect(searchReducer(state, action)).toStrictEqual({
+      ...initialSearchState,
+      offerCategories: ['CINEMA', 'MUSIQUE'],
+      priceRange: [30, 500],
```
Comment (antoinewg) [inline src/features/search/pages/__tests__/reducer.test.ts:70]:
```suggestion
      priceRange: [30, 300],
```

je pense que ça vaut le coup de garder le clampPrice dans `INIT_FROM_SEE_MORE`

File: src/features/home/components/OffersModule.tsx (hunk 1 above)
Comment (kopax-polyconseil) [reply src/features/home/components/OffersModule.tsx]: OK

File: src/features/search/pages/__tests__/reducer.test.ts (hunk 2 above)
Comment (kopax-polyconseil) [reply src/features/search/pages/__tests__/reducer.test.ts]: OK

**Review by antoinewg — APPROVED** [review]

**Pull request merged**